2021-05-20 09:27:37,757 INFO    MainThread:97059 [wandb_setup.py:_flush():69] setting env: {}
2021-05-20 09:27:37,757 INFO    MainThread:97059 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-20 09:27:37,757 INFO    MainThread:97059 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_092737-91jyqo9r/logs/debug.log
2021-05-20 09:27:37,757 INFO    MainThread:97059 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210520_092737-91jyqo9r/logs/debug-internal.log
2021-05-20 09:27:37,758 INFO    MainThread:97059 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd017ef2630>
2021-05-20 09:27:37,758 INFO    MainThread:97059 [wandb_init.py:init():369] calling init triggers
2021-05-20 09:27:37,758 INFO    MainThread:97059 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-20 09:27:37,758 INFO    MainThread:97059 [wandb_init.py:init():391] re-initializing run, found existing run on stack: wjqnfrf3
2021-05-20 09:27:37,759 INFO    MainThread:97059 [wandb_run.py:finish():1208] finishing run ranuga-d/Heart-Disease-UCI/wjqnfrf3
2021-05-20 09:27:37,759 INFO    MainThread:97059 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-20 09:27:37,759 INFO    MainThread:97059 [wandb_run.py:_restore():1507] restore
2021-05-20 09:27:37,763 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 721
}

2021-05-20 09:27:38,315 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 721
}

2021-05-20 09:27:39,175 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 17559
}

2021-05-20 09:27:40,009 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17518
  total_bytes: 17559
}

2021-05-20 09:27:40,112 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17518
  total_bytes: 17559
}

2021-05-20 09:27:40,214 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,316 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,418 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,519 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,621 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,723 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,825 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:40,927 INFO    MainThread:97059 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 17559
  total_bytes: 17559
}

2021-05-20 09:27:42,225 INFO    MainThread:97059 [wandb_run.py:_show_files():1879] logging synced files
2021-05-20 09:27:42,228 INFO    MainThread:97059 [wandb_init.py:init():418] starting backend
2021-05-20 09:27:42,228 INFO    MainThread:97059 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-20 09:27:42,231 INFO    MainThread:97059 [backend.py:ensure_launched():132] starting backend process...
2021-05-20 09:27:42,263 INFO    MainThread:97059 [backend.py:ensure_launched():137] started backend process with pid: 101194
2021-05-20 09:27:42,264 INFO    MainThread:97059 [wandb_init.py:init():423] backend started and connected
2021-05-20 09:27:42,275 INFO    MainThread:97059 [wandb_init.py:init():465] updated telemetry
2021-05-20 09:27:42,275 INFO    MainThread:97059 [wandb_init.py:init():484] communicating current version
2021-05-20 09:27:43,148 INFO    MainThread:97059 [wandb_init.py:init():489] got version response 
2021-05-20 09:27:43,148 INFO    MainThread:97059 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-20 09:27:43,678 INFO    MainThread:97059 [wandb_init.py:init():522] starting run threads in backend
2021-05-20 09:27:48,683 INFO    MainThread:97059 [wandb_run.py:_console_start():1565] atexit reg
2021-05-20 09:27:48,684 INFO    MainThread:97059 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-20 09:27:48,684 INFO    MainThread:97059 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-20 09:27:48,684 INFO    MainThread:97059 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-20 09:27:48,684 INFO    MainThread:97059 [wandb_init.py:init():546] run started, returning control to user process
2021-05-20 09:27:49,959 INFO    MainThread:97059 [wandb_run.py:finish():1208] finishing run ranuga-d/Heart-Disease-UCI/91jyqo9r
